### DISCUSSION TITLE
When is something declared "finished"?

### DISCUSSION THREAD

**Discussion by FredrikAugust** [open-source-ideas/ideas#300] [General]
As of right now, there are a lot of issues that live in a sort of _limbo_ state, where no one feel like starting it, but no one wants to call it done. 

This regularly happens when there exist a lot of other, similar, open source libraries -- leading to the ultimate decline in motivation for starting all over.

Should we close issues that have open source alternatives available when no one throws themselves at the issue?

**Comment by KOLANICH** [comment]
I guess their authors should be asked first. Maybe the alternatives are not what they mean.

**Reply by FredrikAugust** [reply]
I agree. I know there are a lot of issues where several alternatives are posted, but the issue remains stale.

If I have a lot of spare time some day I can try to go through some of them and ask if they can be closed.
